idea/src/org/jetbrains/kotlin/idea/inspections/NullChecksToSafeCallInspection.kt

```diff
@@ -78,7 +81,7 @@ private fun collectNullCheckExpressions(expression: KtBinaryExpression): Triple<
 
 private fun isNullChecksToSafeCallFixAvailable(expression: KtBinaryExpression): Boolean {
     val (lte, rte) = collectNullCheckExpressions(expression) ?: return false
-    return getDuplicates(lte, rte)?.parent as? KtDotQualifiedExpression != null
+    return lte.isChainStable() && getDuplicates(lte, rte)?.parent as? KtDotQualifiedExpression != null
```

Comment:
If i modify last check to `a?.a?.a != null` code will compile, yes, but it didnt test code that checking stability of expression because i explicitly check that is right part of expression is dot, safecall and !! makes intention applicable  `getDuplicates(lte, rte)?.parent as? KtDotQualifiedExpression != null`